src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/TestFiles/CodeDirectiveFormattingTest/CodeBlockDirective_UseTabs.input.html

```diff
@@ -0,0 +1,5 @@
+~~~~~ ~
```

Comment:
Most of the tests in `CodeDirectiveFormattingTest` won't have anything interesting wrt HTML formatting.